columnphysics/icepack_atmo.F90

```diff
@@ -972,24 +941,17 @@ subroutine icepack_atm_boundary(sfctype,                   &
                                    lhcoef,   shcoef,        &
                                    Cdn_atm,                 &
                                    Cdn_atm_ratio_n,         &
-                                   iso_flag = iso_flag,     &
-                                   Qa_iso=l_Qa_iso,         &
-                                   Qref_iso=l_Qref_iso,     &
-                                   uvel=l_uvel, vvel=l_vvel,  &
-                                   Uref=l_Uref, zlvs=zlvs)
+                                   Qa_iso=Qa_iso,           &
+                                   Qref_iso=Qref_iso,       &
```

Comment:
What happens if present(Qref_iso)=F?  Is this robust across all compilers/platforms?  You probably tested that previously...

Reply:
This is robust.  The optional attribute and "present" state is passed down the calling tree.  As long as optional is set in the calling routine for this field if want to check it's "present" state (determined by the original caller) and/or the optional argument is not used unless it exists, all should be fine.  You can definitely get into trouble if you pass the optional argument down the calling tree and use it if it wasn't passed.  Hopefully, we'll avoid that with the logic we have setup.